### PR TITLE
feat(adyen): Added documentation for Magento2 Adyen module version 8

### PR DESCRIPTION
**Pull request by Kagaminara**
[Adyen documentation](https://deploy-preview-440--heuristic-almeida-1a1f35.netlify.app/docs/advanced/payments/adyen)

[Migration Guide](https://deploy-preview-440--heuristic-almeida-1a1f35.netlify.app/docs/appendices/migration-guides/)

### CODE REVIEW

**Comment by Kagaminara** [comment]
[Adyen documentation](https://deploy-preview-440--heuristic-almeida-1a1f35.netlify.app/docs/advanced/payments/adyen)

[Migration Guide](https://deploy-preview-440--heuristic-almeida-1a1f35.netlify.app/docs/appendices/migration-guides/)

**Review by abailleulfc — APPROVED** [review]

File: docs/advanced/payments/adyen.mdx [hunk 1]
```diff
@@ -54,6 +54,9 @@ to integrate with the Adyen Magento module please refer to
 - `FRONT_COMMERCE_ADYEN_STORE_PAYMENT_DETAILS` Boolean (more on
   [Adyen's Stored Payment Methods](#enable-adyens-stored-payment-methods-optional-1)).
   If you want to enable Adyen's Stored Payment Methods (Tokenization).
+- `FRONT_COMMERCE_ADYEN_MAGENTO_MODULE_VERSION` The version of the Magento2
+  Adyen module you use. You can find it in your magento backend, in
+  `Stores > Configuration > Sales > Payment Methods > Adyen Payments > Documentation and Support`.
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:59]: this is under `As a Front-Commerce payment method` while this is for the case where the Magento2 module is used.

also, while writing that, I realize that this environment variable is probably required also in the case where embedded payment is used and that should not be the case. 

and maybe you can add there that we only support the module in version 7 and 8 ?

File: docs/advanced/payments/adyen.mdx [hunk 2]
```diff
@@ -421,6 +425,12 @@ If you are using the default Front-Commerce Adyen component, your customer
 should now be able to save their credit card information during checkout by
 checking the "Save for my next payment" checkbox.
 
+### Enable Paypal payments
+
+If you want to enable PayPal payments when using the Magento2 Adyen module, you
+will need to enable Adyen's `Alternative Payment Methods` located in
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:431]:
```suggestion
will need to enable Adyen's `Alternative Payment Methods` located under
```

(I think)

File: docs/appendices/migration-guides/index.mdx [hunk 3]
```diff
@@ -12,6 +12,15 @@ import ContactLink from "@site/src/components/ContactLink";
...
 import TabItem from "@theme/TabItem";
 
+## `2.17.0` -> `2.18.0`
+
+### Magento2 Adyen module update
+
+In this release, we added support for version 8 of Magento2 Adyen module.
+However, the `FRONT_COMMERCE_ADYEN_MAGENTO_MODULE_VERSION` environment variable
+is now required so you have
+[to configure the environment so that it is defined](/docs/advanced/payments/adyen#Add-the-Adyen-client-key-in-the-environment).
```
Comment (dpobel) [inline docs/appendices/migration-guides/index.mdx:22]:
```suggestion
If you have a Front-Commerce setup where the module `Magento2/Adyen` is used, [the `FRONT_COMMERCE_ADYEN_MAGENTO_MODULE_VERSION` environment variable
is now required and it must contain the Magento2 module version](/docs/advanced/payments/adyen#Add-the-Adyen-client-key-in-the-environment).
```

File: docs/advanced/payments/adyen.mdx (hunk 1 above)
Comment (Kagaminara) [reply docs/advanced/payments/adyen.mdx:59]: > this is under As a Front-Commerce payment method while this is for the case where the Magento2 module is used.

Indeed ! Will update 👍 

> also, while writing that, I realize that this environment variable is probably required also in the case where embedded payment is used and that should not be the case.

As discussed, will add a second config provider to complement the existing one with the new env variable

> and maybe you can add there that we only support the module in version 7 and 8 ?

Will do !

**Review by dpobel — COMMENTED** [review]
:+1:

**Review by dpobel — APPROVED** [review]

**Pull request merged**